.travis.yml

```diff
@@ -8,60 +8,24 @@ sudo: false
 
 matrix:
   include:
-    - env: MONGOPHP=3.0.5.4
+    - env: PHP=5.4
```

Comment:
As the only thing moving in the matrix you could avoid this usage and put only : 

`php:
  - 5.4
  - 5.5
  - 5.6

addons:
  apt:
    packages:
      -libap-tools
`